### ISSUE TITLE
Correct typo in  README.md

### ISSUE THREAD

**Issue by mucahitkantepe** [julien-lafont/protoless#11]
In Quick Start section, it should be  `libraryDependencies` `++=` not `+=`

```
resolvers += Resolver.bintrayRepo("julien-lafont", "maven")

libraryDependencies ++= Seq(
  "io.protoless" %% "protoless-core" % "0.0.6",
  "io.protoless" %% "protoless-generic" % "0.0.6"
)

```

**Comment by mucahitkantepe** [comment]
Also I'm getting 
```
[warn] 	:: io.protoless#protoless-tag_2.12;0.0.6: not found

```
when trying to build my project.

`protoless-tag` is not included in bintray repository: http://dl.bintray.com/julien-lafont/maven/io/protoless/

**Comment by julien-lafont** [comment]
Thanks for the report.

The ivy repository is fixed in the `0.0.7` release!
